commit 24c2db0621a3db2f220815d77b8d6023a6a7d796
Author: WANG Cong <xiyou.wangcong@gmail.com>
Date:   Fri Mar 3 12:21:14 2017 -0800

    strparser: destroy workqueue on module exit
    
    [ Upstream commit f78ef7cd9a0686b979679d0de061c6dbfd8d649e ]
    
    Fixes: 43a0c6751a32 ("strparser: Stream parser for messages")
    Cc: Tom Herbert <tom@herbertland.com>
    Signed-off-by: Cong Wang <xiyou.wangcong@gmail.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1390_24c2db0621a3db2f220815d77b8d6023a6a7d796.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1390_24c2db0621a3db2f220815d77b8d6023a6a7d796.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
